slack_bolt/context/async_context.py

```diff
@@ -133,41 +131,31 @@ async def handle_button_clicks({ACK}, {RESPOND}):
         return self[RESPOND]
 
     @property
-    def complete_success(self) -> AsyncCompleteSuccess:
-        f"""`{COMPLETE_SUCCESS}()` function for this request.
+    def complete(self) -> AsyncComplete:
+        f"""`{COMPLETE}()` function for this request. Once a function's state is set to complete,
+        any outputs the function returns will be passed along to the next step of its housing workflow,
+        or complete the workflow if the function is the last step in a workflow. Additionally,
+        any interactivity handlers associated to a function invocation will no longer be invocable.
 
             @app.function("reverse")
             async def handle_button_clicks(context):
-                await context.{COMPLETE_SUCCESS}({{"stringReverse":"olleh"}})
+                await context.{COMPLETE}(outputs={{"stringReverse":"olleh"}})
 
             @app.function("reverse")
-            async def handle_button_clicks({COMPLETE_SUCCESS}):
-                await {COMPLETE_SUCCESS}({{"stringReverse":"olleh"}})
+            async def handle_button_clicks({COMPLETE}):
+                await {COMPLETE}(outputs={{"stringReverse":"olleh"}})
 
-        Returns:
-            Callable `{COMPLETE_SUCCESS}()` function
-        """
-        if COMPLETE_SUCCESS not in self:
-            self[COMPLETE_SUCCESS] = AsyncCompleteSuccess(
-                client=self.client, function_execution_id=self.function_execution_id
-            )
-        return self[COMPLETE_SUCCESS]
-
-    @property
-    def complete_error(self) -> AsyncCompleteError:
-        f"""`{COMPLETE_ERROR}()` function for this request.
+        Args:
+            ** THIS FUNCTION DOES NOT ACCEPT POSITIONAL ARGUMENTS **
 
-            @app.function("reverse")
-            async def handle_button_clicks(context):
-                await context.{COMPLETE_ERROR}("an error spawned")
-
-            @app.function("reverse")
-            async def handle_button_clicks({COMPLETE_ERROR}):
-                await {COMPLETE_ERROR}("an error spawned")
+        Kwargs:
+            ** THIS FUNCTION ONLY ACCEPTS ONE OF THE KWARGS BELOW **
```

Comment:
We should check if this additional statement line works as you expect in the API documents. You can generate https://slack.dev/bolt-python/api-docs/slack_bolt/ on your local machine by running `./scripts/generate_api_docs.sh` @WilliamBergamin Can you check this?

Reply:
Good point, I ended up moving this information to `slack_bolt/context/complete/complete & async_complete` in order for the doc string to be visible in IDE previews. (tested with vs code)